readme.md

```diff
@@ -35,7 +35,36 @@ grunt.initConfig({
 grunt.loadNpmTasks('grunt-eslint');
 grunt.registerTask('default', ['eslint']);
 ```
+### [gulp-eslint](https://github.com/adametry/gulp-eslint)
+```js
+var gulp = require('gulp');
+var eslint = require('gulp-eslint');
+
+gulp.task('lint', function() {
+	return gulp.src('./example.js')
```

Comment:
no need to use return when arrow function

see: https://github.com/sindresorhus/gulp-ava#usage